monarchmoney/monarchmoney.py

```diff
@@ -1864,6 +1869,28 @@ async def set_budget_amount(
             graphql_query=query,
         )
 
+    async def upload_account_balance_history(
+        self, account_id: str, csv_content: str, filename: str = "upload.csv"
+    ) -> None:
+        """
+        Uploads the account balance history csv for the given account.
+
+        :param account_id: The account ID to upload the balance history.
```

Comment:
It doesn't appear to have any impact in the product. I believe its in the request as an identifier in case you want to make a batch call with multiple files and map them to corresponding accounts. I'll remove the parameter to avoid the ambiguity.